### ISSUE TITLE
COVID-19 Vaccine Trials Content and Tooling - Accessibility Feedback - Usability Test Prep

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#11990]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Must
* Ensure the height and weight inputs have clear labeling and each input has at least one label mapped to it programmatically. Happy to meet any time to discuss design, code, ad hoc testing of the interface.

## Consider
* Pinging a11y specialists to test code in progress for the form. Since it's a public form, we should be able to test locally on a feature branch. Trevor can do some testing on the MacOS in a preview instance as well.

## Definition of Done
* Team has responded with answers to the Must and Should questions. These can be 2-3 sentence summaries, links to prototypes / live code, whatever the team feels are the best ways to respond.

**Comment by batemapf** [comment]
@1Copenut we're just about ready for another round of review. Probably Friday or Monday. Is there another ticket I need to create for that or just keep using this one?

**Comment by 1Copenut** [comment]
@batemapf Hi Patrick. This document will be fine for the next round of review. Are you looking for a screen reader spot check, or something else?
